src/main/kotlin/uk/gov/justice/digital/assessments/api/DisabilityAnswerDto.kt

```diff
@@ -28,7 +29,11 @@ class DisabilityAnswerDto(
         code = deliusDisability.disabilityType.code,
         description = deliusDisability.disabilityType.description,
         notes = deliusDisability.notes,
-        adjustments = deliusDisability.provisions?.map { it.provisionType?.description }
+        adjustments = deliusDisability.provisions?.filter {
+          val now = LocalDate.now()
```

Comment:
Can we separate this filter logic into a separate function call to keep it cleaner?